apps/backoffice-v2/src/common/hooks/useRadixScrollBoundaries/useRadixScrollBoundaries.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useRef } from 'react';
+
+export const useRadixScrollBoundaries = () => {
+  const scrollAreaRef = useRef<HTMLDivElement>(null);
+
+  const resetScrollPosition = () => {
+    localStorage.removeItem('scrollPosition');
+  };
+
+  const restoreScrollPosition = () => {
+    const savedPosition = localStorage.getItem('scrollPosition');
+
+    if (savedPosition && scrollAreaRef.current) {
+      if (scrollAreaRef.current) {
+        scrollAreaRef.current.scrollTop = parseInt(savedPosition, 10);
```

Comment:
Not a must, consider using the scroll method over mutating scrollTop. It's subjective unless we want to use scroll behavior.